public/js/components/Editor.js

```diff
@@ -49,6 +49,9 @@ const Editor = React.createClass({
       gutters: ["breakpoints"]
     });
 
+    // NOTE: only used for testing
+    window.cm = this.editor;
```

Comment:
I'd rather avoid littering these around our code, as mentioned in the other places as well. I'm pretty sure you can get the CodeMirror instance from the DOM element that it sits in. That's a lot cleaner than this, so we should look into doing that instead.

I don't see it in the docs off-hand, but I'll look more. You can merge this but I'll file an issue to fix this.